setup.cfg

```
@@ -73,7 +73,7 @@ cuda100 =
     cupy-cuda100>=5.0.0b4
 # Language tokenizers with external dependencies
 ja =
-    mecab-python3==0.7
+    fugashi>=0.1.3
```

We always want to have closed ranges, or eventually this version of the library would be uninstallable:
```suggestion
    fugashi>=0.1.3,<0.2.0
```

---

I'll fix this post-merge, I don't want to block the PR.

---

Thanks, and understood, I'll keep that in mind next time.